packages/shared/src/telemetry/attributes.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * The attributes of an OpenTelemetry compliant event for flag evaluation.
+ * @see https://opentelemetry.io/docs/specs/semconv/feature-flags/feature-flags-logs/
+ */
+export const TELEMETRY_ATTRIBUTE = {
```

Comment:
I still think constants like `export const TELEMETRY_ATTRIBUTE_FEATURE_FLAG_KEY` are better for minification and tree shaking.

Reply:
I made the switch but some of the the names are so long that they don't fit in the intellisense dropdown.

Reply:
It's up to you. It's your library. This is just the style we use in otel for bundle size reasons.

Reply:
I reverted back after talking to @toddbaert. We agreed that the ergonomic improves outweigh the slight tree shaking penalty.